examples/showVisitSkyMap.py

```diff
@@ -99,21 +114,34 @@ def main(rootDir, tract, visits, ccds=None, ccdKey='ccd', showPatch=False, saveF
 
     # draw the skymap
     if showPatch:
-        skymap = butler.get('deepCoadd_skyMap', {'tract': 0})
-        for tract in skymap:
-            for patch in tract:
-                ra, dec = bboxToRaDec(patch.getInnerBBox(), tract.getWcs())
-                pyplot.fill(ra, dec, fill=False, edgecolor='k', lw=1, linestyle='dashed')
-                if xlim[1] < percent(ra) < xlim[0] and ylim[0] < percent(dec) < ylim[1]:
-                    pyplot.text(percent(ra), percent(dec, 0.9), str(patch.getIndex()),
-                                fontsize=6, horizontalalignment='center', verticalalignment='top')
+        alpha0 = 1.0
+        for i_t, tract in enumerate(tracts):
+            alpha = max(0.1, alpha0 - i_t*1.0/len(tracts))
+            skymap = butler.get('deepCoadd_skyMap', {'tract': tract})
```

Comment:
I'm not sure if this data ID is used at all; doesn't Butler simply discard it? If so maybe it can be outside of the for loop?